### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by jpmckinney**
Creating PR so that this branch and these commits are better surfaced on GitHub.

### CODE REVIEW

File: standard/docs/en/implementation/serialization.md [hunk 1]
```diff
@@ -92,7 +92,9 @@ Whilst this allows complex data to be expressed in flat CSV, users will need to
 
 Instead, data with a one-to-many relationship can be represented using a multi-table serialization. 
 
-Note: OCDS CSV files MUST only be encoded in either UTF-8
+Note
```
Comment (jpmckinney) [inline standard/docs/en/implementation/serialization.md]: @duncandewhurst Was the intention to use a `.. note::` admonition?

Comment (duncandewhurst) [reply standard/docs/en/implementation/serialization.md]: I didn't intend to change the formatting, but yes it makes sense for this note to be in a suitable admonition - I'll add a commit to this branch

**Comment by jpmckinney** [comment]
The only notable change is https://github.com/open-contracting/standard/pull/726/commits/7a84bbbbb2ade6f560475fb7774dd92ea307faaf

**Comment by jpmckinney** [comment]
My additional commits were only to get tests to pass (by running make_validation_schema.py), to avoid interaction with Transifex (Replicate typo fixes in PO files), and to update the changelog. Merging.

**Pull request merged**